v0/destinations/bqstream/transform.js

```diff
@@ -0,0 +1,62 @@
+/* eslint-disable no-console */
+const PodCache = require("../../../cache/pod-cache");
+const {
+  defaultRequestConfig,
+  defaultPostRequestConfig
+} = require("../../util");
+
+
+const responseWrapper = response => {
+  const resp = defaultRequestConfig();
+  resp.endpoint = response.endPoint;
+  resp.method = defaultPostRequestConfig.requestMethod;
+  resp.headers = { ...response.headers, "Content-Type": "application/json" };
+  resp.body.JSON = response.payload;
+  return resp;
+};
+
+function process(event) {
+  const { message } = event;
+  const { properties } = message;
+  const {
+    destination: {
+      Config: { datasetId, tableId }
+    }
+  } = event;
+
+  const payload = {
+    rows: [
+      {
+        json: { ...properties }
+      }
+    ]
+  };
+  return responseWrapper({
+    payload,
+    method: "POST",
+    // TODO: ProjectID(rudder-sai) can be referred in a more customised way!
+    endPoint: `https://bigquery.googleapis.com/bigquery/v2/projects/rudder-sai/datasets/${datasetId}/tables/${tableId}/insertAll`
```

Comment:
did you miss addressing this TODO?

Reply:
Also, `rudder-sai` sounds informal if we were to keep it

Reply:
Not missed, I would say rather wanted to discuss it before merging. `rudder-sai` was intended for local-dev purposes. That wouldn't be the case basically. May be even handle `projectId` similar to how `datasetId` & `tableId` are also currently being handled. It should be a very small change